src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1375,6 +1377,23 @@ public function getRemovedIds(): array
 EOF;
     }
 
+    private function addDeprecatedParameters(): string
+    {
+        $deprecated = $this->container->getParameterBag()->allDeprecated();
+        if (!$deprecated) {
```

Comment:
`if (!$deprecated = $this->container->getParameterBag()->allDeprecated()) {`

Reply:
Done.